swagger.yml

```diff
@@ -512,7 +565,25 @@ definitions:
       - height
       - unit
 
-  Packages:
+  WeightOptions:
+    title: WeightOptions
+    type: object
+    description: The weight of the package. Orders will be classified as `HEAVY` if any of its package are over 50lbs.
+    properties:
+      weight:
+        type: integer
+        format: int32
+        description: The weight of the package
+        example: 20
+      unit:
+        type: string
+        description: The unit of weight for the package. Can be either `lb` or `kg`.
+        example: lb
+    required:
+      - weight
+      - unit
+
+  UpdateArrayOfPackages:
```

Comment:
Definitions `CreateArrayOfPackages`, `UpdateArrayOfPackages`, and `Packages` differ in that the package definition for create does not have a tracking number, while the update one does have one. Moreover, the `Packages` definition (the result of GETs) also has the package status.